### ISSUE TITLE
tsfresh returns an empty feature dataframe also with enough samples

### ISSUE THREAD

**Issue by muscionig** [blue-yonder/tsfresh#442]
Hi,
I am working on windows, tsfresh version 0.11.1.
I have a dataset of 13 series per observations, each series has 20001 samples, that is the reason I am thinking the sample size is enough. These series are not trivial, they are not constant and they do not have a trivial period value.

I am using the feature extraction tools, however, I am getting no features from it.

``` python
for serie_name in df_series:
        df_to_pivot = pd.DataFrame()
        df_to_pivot = df[['TIME', serie_name]]
        df_to_pivot.columns = ['TIME', 'Event']
        df_to_pivot.loc[:,'ID'] = file_name+'_'+serie_name        
        df_to_pivot_label = {}
        df_to_pivot_label['ID'] = file_name+'_'+serie_name
        try:
            df_to_pivot_label['LABEL'] = int(df_label.loc[serie_name,:].values[0])
        except ValueError:
            df_to_pivot_label['LABEL'] = df_label.loc[serie_name,:].values[0]
            
        print(df_to_pivot_label)
            
        df_label_pivot.append(df_to_pivot_label)
        df_pivot =df_pivot.append(df_to_pivot)        
        df_label_pivot = pd.DataFrame(df_label_pivot)
        df_label_pivot.set_index('ID', inplace=True)
        df_label_pivot = df_label_pivot['LABEL']
        extracted_features = pd.DataFrame()
        extracted_features1 = extract_features(df_pivot, column_id='ID',column_sort="TIME", n_jobs=0)
        extracted_features2 = extract_relevant_features(df_pivot, df_label_pivot, column_id='ID',column_sort="TIME", n_jobs=0, fdr_level=0.95)
```
    
That is the essential part of the code and attached a sample of the dataset I am using.
[sample_data.zip](https://github.com/blue-yonder/tsfresh/files/2454310/sample_data.zip)

I think there is an error as I can extract simple meaningful statistics (mean, std, etc).

**Comment by nikhase** [comment]
Please try specifying the `column_value` parameter in `extract_features` explicitly with the column name that contains the time series and give an update.

**Comment by muscionig** [comment]
Same behavior:

new line: `extracted_features_d = extract_relevant_features(df_pivot, df_label_pivot, column_id='ID',column_sort="TIME", n_jobs=0, column_value='Event')`

return: `WARNING:tsfresh.feature_selection.relevance:No feature was found relevant for classification for fdr level = 0.05. Consider using a lower fdr level or other features.`

**Comment by nikhase** [comment]
Did you try it with `extract_features` as well or only `extract_`**relevant**`_features`?

**Comment by muscionig** [comment]
I am trying with both, extract_features returns me feasible values, but I would like to statistically prove the significance of these values.

**Comment by nikhase** [comment]
The warning message indicates that there is no statistically significant feature. You might need to try other features.

**Comment by muscionig** [comment]
Yes, but if I use the features in a different test, like the one available in sklearn.feauture_selection (as SelectKBestFeatures) there is a correlation, I would say a pretty strong correlation.

**Comment by nikhase** [comment]
A quick glance at your provided data shows that the time series values barely change. I think that tsfresh cannot get any significant features from this. I think it is a data problem. 
Also I don't know which correlation from sklearn you are referring to.

**Comment by nikhase** [comment]
I am sorry, as this seems to be a data problem, I cannot help you with that. And I also cannot make any sense of this data provided as I see no classes as well.
A last guess is: maybe try standardization of the time series first and see if this changes the results.

**Comment by muscionig** [comment]
I cannot send the label as they are private now, I am wondering on the statistical test that tsfresh is doing on the data. Features_selection of sklearn is a well-known and used class, so if it returns some good scores for the features and tsfresh not I think there may be a problem, anyway I will implement my significance test.

**Comment by MaxBenChrist** [comment]
@nikhase I don't think that standardisation will change the tsfresh feature selection results much. Our used hypothesis tests are nonparametric, they don't make any assumptions about the distribution of the features. 

This would be an interesting investigation, how much the selected features change if we standardize.

**Comment by nils-braun** [comment]
Seems this issue is stalled. Closing now.

**Comment by wobuhuibei** [comment]
have the same problem. Have you solved it now? @muscionig
